### PR TITLE
Grammar clean up

### PR DESCRIPTION
**Pull request by tone81**
@tone81 is being a grammar nazi and is trying to clean it up without losing the intent.

* Removed any sentence fragments.
* Single spacing between sentences
* Might look at passive verbs and fix them

### CODE REVIEW

**Comment by tone81** [comment]
@OronNadiv @lanetix/engineers Looking to merge grammar fixes before content changes.

File: _posts/2015-10-19_hiring_process.markdown [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+Most people will agree that the “standard” interview process for engineers is broken. It is important to not only know how to implement linked list, but also deliver high-quality code on time. A successful engineer may or may not know five different implementations of the singleton pattern but MUST be a team player and have good communication skills. 
 
-These are the qualities we seek in our successful engineers:
+These are the qualities we seek in our engineers:
 
-* Resourcefulness / Problem Solving: Your job as an engineer is to solve bugs, implement features and design architecture.  We want our engineers to search for creative solutions.  An important part of being resourceful is asking peers for help.
+* Resourcefulness / Problem Solving: Your job as an engineer is to solve bugs, implement features, and architect solutions. We want our engineers to search for creative solutions. An important part of being resourceful is asking peers for help.
```
Comment (apechimp) [inline _posts/2015-10-19_hiring_process.markdown]: :-1: oxford comma :trollface:

Comment (tone81) [inline _posts/2015-10-19_hiring_process.markdown]: Throwing the :book: about Oxford commas at you!

File: _posts/2015-10-19_hiring_process.markdown [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-* Passion: Be excited about what you do!  Many of our engineers code outside of working hours.  Whether it is by contributing to open source or a little side project,  we want to work with passionate engineers looking to grow both personally and professionally.
+* Passion: Be excited about what you do!  Many of our engineers code outside of working hours. Whether it is by contributing to open source or a little side project, we want to work with passionate engineers looking to grow both personally and professionally.
 
-* Self managed:  Effective time management.  We expect our engineers to complete tasks prior to the established deadlines. 
+* Self managed: We expect our engineers to manage their time effectively and to complete tasks prior to the established deadlines. 
 
 Unimportant criteria:
 
-* Knowledge or experience in a specific programming language / tools / technology; Intelligent, ambitious engineers can learn new technologies quickly.
+* Knowledge or experience in a specific programming language / tools / technology - Intelligent, ambitious engineers can learn new technologies quickly.
```
Comment (apechimp) [inline _posts/2015-10-19_hiring_process.markdown]: These should all be [emdashes](http://www.thepunctuationguide.com/em-dash.html).

File: _posts/2015-10-19_hiring_process.markdown [hunk 3]
```diff
@@ -1,35 +1,35 @@
...
 
 We came up with three different options:
 
-1. Giving a 45 minute presentation about a topic of your choice.  It may be a project you did in the past.  It can also be an interesting technology you recently discovered.  We want you to share your thoughts and experience with us through a conversation.
+1. Giving a 45 minute presentation about a topic of your choice. It may be a project you did in the past. It can also be an interesting technology you recently discovered. We want you to share your thoughts and experience with us through a conversation.
 
-2. Complete an assignment in your free time.  Some people perform better in a real life environment.  Our candidate can pick one of the six different assignments to take home.  Once an assignment is done, one of our engineers will spend time with the candidate to review the architecture and the implementation details.
+2. Complete an assignment on your free time. Some people perform better in a real life environment. Our candidate can pick one of the six different assignments to take home. Once an assignment is done, one of our engineers will spend time with the candidate to review the architecture and the implementation details.
```
Comment (apechimp) [inline _posts/2015-10-19_hiring_process.markdown]: If we're gonna revisit wording, consider 'Do a homework assignment.' Who has free time? :wink:

File: _posts/2015-10-19_hiring_process.markdown (hunk 2 above)
Comment (apechimp) [inline _posts/2015-10-19_hiring_process.markdown]: —

**Comment by apechimp** [comment]
:shipit:

**Pull request merged**